packages/next-swc/crates/napi/src/turbo_trace_server.rs

```diff
@@ -0,0 +1,17 @@
+use std::{path::PathBuf, thread};
+
+use napi::JsBoolean;
+
+#[napi]
+pub fn start_turbopack_trace_server(path: String, in_process: JsBoolean) -> napi::Result<String> {
```

Comment:
```suggestion
pub fn start_turbopack_trace_server(path: String, in_process: bool) -> napi::Result<String> {
```

I think that should work too

Reply:
I thought so too but then it failed at runtime, so had to change to JsBoolean 😄